features/admin.organizations.v1/components/organization-list.tsx

```diff
@@ -488,7 +488,8 @@ export const OrganizationList: FunctionComponent<OrganizationListPropsInterface>
                 <EmptyPlaceholder
                     className={ !isRenderedOnPortal ? "list-placeholder mr-0" : "" }
                     action={
-                        onEmptyListPlaceholderActionClick && (
+                        isFeatureEnabled(featureConfig.organizations, "organizations.create")
```

Comment:
Let's define the feature identifiers in a FEATURE_DICTIONARY object in constants folder and use it here.